src/pages/Home.js

```diff
@@ -12,8 +12,30 @@ export default function Home(props) {
   function handleClick() {
     const token = getToken();
     addKeyValuePairToLocalStorage('token', token);
+    db.collection(token).add({
+      item_name: 'Your Item',
+      last_purchased: 'Soon, Later, Whenever',
+      purchase_frequency: null,
+    });
```

Comment:
I love that you got creative here! I totally understand initializing the token with an "empty" object. So that the list persists when sharing the token with a friend, etc. But, I think that we will have to implement a different solution eventually. Maybe going with @yenly's suggestion to have a separate collection of just initialized tokens? 

I'm not sure what the optimal solution is, but I think a user might be confused by an empty item in their list.

Reply:
Thank you! But yes, on the same page as you for sure, James. 

Yenly did a nice job of explaining how the choices made were not quite what should be done, given the acceptance criteria from previous weeks as well as this week. I'll admit liking my idea of the initial list object to be an example for the user, but I bet it could also be annoying pretty quickly for the user, especially if they are a frequent user of the product. 

Also, the idea is currently in conflict with this week's AC, so that's top consideration! 😃